tdrs-backend/tdpservice/users/models.py

```diff
@@ -42,6 +42,199 @@ class RegionMeta(models.Model):
     )
 
 
+class UserChangeRequestStatus(models.TextChoices):
+    """Enum of options for change request status."""
+
+    PENDING = "pending", _('Pending')
+    APPROVED = "approved", _('Approved')
+    REJECTED = "rejected", _('Rejected')
+
+
+class UserChangeRequest(models.Model):
+    """Model to track user information change requests."""
+
+    class Meta:
+        """Define meta attributes."""
+
+        ordering = ['-requested_at']
+        verbose_name = _('User Change Request')
+        verbose_name_plural = _('User Change Requests')
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    user = models.ForeignKey(
+        'users.User',
+        on_delete=models.CASCADE,
+        related_name='change_requests'
+    )
+    requested_by = models.ForeignKey(
+        'users.User',
+        on_delete=models.CASCADE,
+        related_name='requested_changes'
+    )
+    field_name = models.CharField(
+        max_length=100,
+        help_text=_('The name of the field being changed')
+    )
+    current_value = models.TextField(
+        blank=True,
+        help_text=_('The current value of the field')
+    )
+    requested_value = models.TextField(
+        blank=True,
+        help_text=_('The requested new value for the field')
+    )
+    status = models.CharField(
+        max_length=20,
+        choices=UserChangeRequestStatus.choices,
+        default=UserChangeRequestStatus.PENDING,
+        help_text=_('The current status of this change request')
+    )
+    requested_at = models.DateTimeField(
+        auto_now_add=True,
+        help_text=_('When this change was requested')
+    )
+    reviewed_at = models.DateTimeField(
+        null=True,
+        blank=True,
+        help_text=_('When this change request was reviewed')
+    )
+    reviewed_by = models.ForeignKey(
+        'users.User',
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='reviewed_changes',
+        help_text=_('The admin who reviewed this change request')
+    )
+    notes = models.TextField(
+        blank=True,
+        help_text=_('Admin notes on approval/rejection')
+    )
+
+    def __str__(self):
+        """Return string representation."""
+        return f"{self.user.username} - {self.field_name} - {self.get_status_display()}"
+
+    def approve(self, admin_user, notes=None):
+        """Approve the change request and apply changes to the user."""
```

Comment:
The change request can submit multiple changes at the same time. 
1. One approach is to check if there is already a change request and block new one if there is one existing
2. or we can accept the new change request and check for previous value vs the change request, if the previous value doesn't match, then do not apply the change. However, I think it is better to reject new change request if there is one already existing